src/cellflow/metrics/_metrics.py

```diff
@@ -113,16 +204,46 @@ def maximum_mean_discrepancy(x: ArrayLike, y: ArrayLike, gamma: float = 1.0, exa
     return xx.mean() + yy.mean() - 2 * xy.mean()
 
 
-def compute_scalar_mmd(x: ArrayLike, y: ArrayLike, gammas: float | None = None) -> float:
-    """Compute MMD across different length scales"""
+def compute_scalar_mmd(x: ArrayLike, y: ArrayLike, gammas: Sequence[float] | None = None) -> float:
+    """Compute the Mean Maximum Discrepancy (MMD) across different length scales
+
+    Parameters
+    ----------
+        x
+            An array of shape [num_samples, num_features].
+        y
+            An array of shape [num_samples, num_features].
+        gammas
+            A sequence of values for the paramater gamma of the rbf kernel.
+
+    Returns
+    -------
+        A scalar denoting the average MMD over all gammas.
+    """
     if gammas is None:
         gammas = [2, 1, 0.5, 0.1, 0.01, 0.005]
     mmds = [maximum_mean_discrepancy(x, y, gamma=gamma) for gamma in gammas]  # type: ignore[union-attr]
     return np.nanmean(np.array(mmds))
 
 
 def compute_metrics_fast(x: ArrayLike, y: ArrayLike) -> dict[str, float]:
```

Comment:
can we expose it, but not add to the docs? to keep it for backward compatibility ,but it's a bit redundant.